### PR TITLE
Web: Make s3 fields optional

### PR DESCRIPTION
**Pull request by kimlisa**
requires 
- [x] https://github.com/gravitational/teleport/pull/39912
- [x] e2e testing

Description:
- When creating AND editing integration: allow s3 fields to be optional, if user leaves it empty, render a warning
- When listing integrations without s3 fields defined, renders a tooltip that provides information to the user on how to update their thumbprint should they run into issues

listing:
image

listing, after clicking on button:
image

editing:
image

creating:
image

changelog: Amazon S3 fields are now optional when creating or editing AWS OIDC integration on the web UI

### CODE REVIEW

**Review by avatus — APPROVED** [review]
I didn't get to run it e2e yet but code looks good

File: web/packages/design/src/Alert/Alert.jsx [hunk 1]
```diff
@@ -56,8 +56,16 @@ const kind = props => {
       };
     case 'outline-info':
       return {
-        background: fade(theme.colors.link, 0.1),
-        border: `${theme.radii[1]}px solid ${theme.colors.link}`,
+        background: fade(theme.colors.accent.main, 0.1),
+        border: `${theme.radii[1]}px solid ${theme.colors.accent.main}`,
```
[the remark this answers is not in the document]
Comment (kimlisa) [reply web/packages/design/src/Alert/Alert.jsx:60]: what is the use case for radii then?

[the remark this answers is not in the document]
Comment (avatus) [reply web/packages/design/src/Alert/Alert.jsx:60]: It should be specific to `border-radius`. Also, I didn't see this before, but our theme has `theme.borders` so we could just do 
```suggestion
        border: `${theme.borders[2]} ${theme.colors.accent.main}`,
```

[check the sharedStyles here](https://github.com/gravitational/teleport/blob/master/web/packages/design/src/theme/themes/sharedStyles.ts#L53)

**Comment by kimlisa** [comment]
friendly ping @rudream (i'll apply michaels change right before i merge)

**Review by rudream — APPROVED** [review]

**Comment by codingllama** [comment]
FYI, this broke `make lint-license`, which breaks Go PRs like #40337. It's likely that the linter isn't always running when it should.

**Pull request merged**